style.css

```diff
@@ -752,3 +746,58 @@ textarea {
     font-size: 87.5%;
   }
 }
+
+.close-x {
+  display: flex;
+  justify-content: end;
+  font-size: 57px;
+  width: 92%;
+}
+
+.nav-list {
+  display: none;
+  right: 0;
+  bottom: 0;
+  top: 0;
+  left: 0;
+  position: absolute;
+  background: #6070ff;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(5px);
+  color: #fff;
+  height: 993px;
+  flex-direction: column;
+  font-size: 20px;
+  gap: 20px;
+  padding-left: 30px;
+}
```

Comment:
- [OPTIONAL ] Great job 👍🏻  but what if you used `position: fix` instead of `absolute`? This will prevent your mobile navigation menu from scrolling, kindly this this


- [OPTIONAL ] It would be better for the background not to be scrollable when the mobile nav menu is open . you can use the CSS property `overflow: hidden;` to prevent that.

- [ OPTIONAL] I think  it will be better as well to close the mobile navigation menu page when the user clicks any option of the mobile menu. I highly recommend you do this in the meantime, I make it optional since the previous reviewer is skipping this.

- [ ]  Adding these features will make your design look great and look exactly like the Figma design,